static/css/perdiem.css

```diff
@@ -276,3 +276,12 @@ form > input[type=submit] {
     display: block;
     margin: 0 auto;
 }
+
+ul.tabs {
+    text-align: center;
+}
+
+ul.tabs li {
+    float: none !important;
```

Comment:
Actually, it works without !important. I couldn't get it to work and saw that on a foundation forum and it worked, but seems to be working without it as well. Re-commiting